pkg/elle/txn/wr_txn.go

```diff
@@ -0,0 +1,162 @@
+package txn
+
+import (
+	"fmt"
+
+	"k8s.io/apimachinery/pkg/util/rand"
```

Comment:
seriously?

Reply:
Emmm I write randInt, should i change it?